Competition2018/src/main/java/competition/commandgroups/DisengageWinchAndReleasePawlCommandGroup.java

```diff
@@ -3,20 +3,22 @@
 import com.google.inject.Inject;
 
 import competition.subsystems.climb.commands.AscendClimberCommand;
+import competition.subsystems.climb.commands.AscendLowPowerCommand;
 import competition.subsystems.climb.commands.DecendClimberCommand;
 import competition.subsystems.climb.commands.ReleasePawlCommand;
 import edu.wpi.first.wpilibj.command.CommandGroup;
+import xbot.common.command.BaseCommandGroup;
 
-public class DisengageWinchAndReleasePawlCommandGroup extends CommandGroup{
+public class DisengageWinchAndReleasePawlCommandGroup extends BaseCommandGroup {
     
     @Inject
     public DisengageWinchAndReleasePawlCommandGroup(
-            AscendClimberCommand ascend,
+            AscendLowPowerCommand ascendLowPower,
             ReleasePawlCommand release,
             DecendClimberCommand decend) 
     {
-        this.addParallel(release);
-        this.addSequential(ascend, 0.1);
+        this.addSequential(release);
+        this.addSequential(ascendLowPower, 0.05);
```

Comment:
This is meant to just ever so briefly pull in the strap, and it's worked really well.